opennms-services/src/main/java/org/opennms/netmgt/enlinkd/snmp/LldpLocalGroupTracker.java

```diff
@@ -92,10 +93,10 @@
     public static String getDisplayable(final SnmpValue snmpValue) {
         String decodedsnmpValue = snmpValue.toHexString();
         try {
-            if (snmpValue.isDisplayable())
+            if (AbstractSnmpValue.allBytesUTF_8(snmpValue.getBytes()))
```

Comment:
So basically we are limiting the displayable to UTF-8 validation. 
If Enlinkd expects this way, that's fine. 
Otherwise I'm good with skipping the test validation for values in order to get the original fix of preventing `ArrayIndexOutOfBoundsException`

Reply:
I could still make it fall back to trying ASCII at least.  Though in this case that's still a single `ï` character, which also seems wrong -- I suspect these devices have a bug in their "null entry" responses that cause this.

Reply:
It _used_ to return `cf` though, so treating it as UTF-8 text seems to keep the same behavior here.

Reply:
I'm open to either way of doing it, if you have a preference.

Reply:
Let's go with this one.